Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1939,6 +1934,7 @@ extension BrowserViewController: TabManagerDelegate {
             // we therefore have to hide all the scrollViews that we are no actually interesting in interacting with
             // to ensure that scrollsToTop actually works
             wv.scrollView.hidden = true
```

Comment:
would be worth testing, but if we are removing the `webView` we no longer need to hide the scroll view in order to enable `scrollToTop`, so this line can be deleted